docs/source/developer_guides/custom_models.mdx

```diff
@@ -0,0 +1,194 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Working with custom models
+
+## LoRA
```

Comment:
I think we can remove this section header since there aren't any other fine-tuning techniques here.

Reply:
I think there will be more in the future, because other techniques such as IA³ should also work with custom models. If you prefer, I can still remove it now and then add it back in later.

Reply:
Yeah, I think it'd be nicer if we structure it around the content we have right now, and it'd be easy to add it back in later when we have more content!

Reply:
Done